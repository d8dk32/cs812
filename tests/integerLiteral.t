

int main() {
  out 042;
  out 0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000042;
  out 123456789;
}

