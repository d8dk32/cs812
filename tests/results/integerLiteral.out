42
42
123456789
